server/internal/news/index.ts

```diff
@@ -129,7 +129,7 @@ class NewsManager {
       where: { id },
     });
     if (article.image) {
-      return await objectHandler.delete(article.image);
+      return await objectHandler.deleteWithPermission(article.image);
```

Comment:
This need to bypass permissions

Reply:
the underlying model iirc bypasses if no user is provided. i might have read the control flow wrong, but thats what it looked like happened

Reply:
If no userId is passed, it treats the request as 'anonymous'

Reply:
ahhh, i'll create a permissionless method then